pkg/reconciler/v1alpha1/configuration/resources/build_test.go

```diff
@@ -102,9 +102,6 @@ func TestBuilds(t *testing.T) {
 				}},
 			},
 			"status": map[string]interface{}{
-				"startTime":      nil,
```

Comment:
Why are these 3 fields removed with the v0.3.0 bump? It looks like these fields including the 4th stepsCompleted were marked as "+optional", but are still present in the status.

Reply:
@dgerd yep, initially I thought the 4 would go… :sweat_smile: I'll look into it :wink:

Reply:
`omitempty` was meaningless without the pointer.  They are pointers now.